cli/npm/installer/local.rs

```diff
@@ -134,37 +133,22 @@ impl NpmPackageFsInstaller for LocalNpmPackageInstaller {
     };
     sync_resolution_with_fs(
```

Comment:
We should probably inline this function call to prevent the verbosity of all the `self.x` stuff. I'll do it in a separate PR though.